Index: testdbt.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- testdbt.py	(revision 06e605f483d84f46449bc30c31a3d6381d4ff6bf)
+++ testdbt.py	(date 1568964058663)
@@ -11,7 +11,7 @@
     html.H1(children='NEW TRY'),
 
     html.Div(children='''
-        Dash: A web application framework for Python.
+        Trying to learn DASH
     '''),
 
     dcc.Graph(
